<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Suite]">
  <listeners>
    <listener class-name="com.automation.Generic.Listener"/>
  </listeners>
  <test thread-count="5" name="BookFlightTest(failed)">
    <classes>
      <class name="com.automation.Test.BookingTest">
        <methods>
          <include name="beforeClass"/>
          <include name="turnOffWifi"/>
          <include name="afterTest"/>
          <include name="beforeTest"/>
        </methods>
      </class> <!-- com.automation.Test.BookingTest -->
    </classes>
  </test> <!-- BookFlightTest(failed) -->
  <test thread-count="5" name="loginTest(failed)">
    <classes>
      <class name="com.automation.Test.LoginTest">
        <methods>
          <include name="beforeClass"/>
          <include name="verifyLoginWithInValidData" invocation-numbers="0 1 2"/>
          <include name="afterTest"/>
          <include name="beforeTest"/>
        </methods>
      </class> <!-- com.automation.Test.LoginTest -->
    </classes>
  </test> <!-- loginTest(failed) -->
</suite> <!-- Failed suite [Suite] -->
